src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,9 +17,22 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.0.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.0.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
     <PackageReference Include="System.Text.Json" Version="4.6.0" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="5.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection.Abstractions" Version="5.0.7" />
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="5.0.7" />
```

Comment:
I don't believe 5.x packages are expected to be used with netcoreapp3.1. The 3.x train of packages are still maintained side-by-side with 5.x https://www.nuget.org/packages/Microsoft.AspNetCore.DataProtection/6.0.0-preview.5.21301.17

Reply:
Also, 2.x is still supported until August 21. https://dotnet.microsoft.com/platform/support/policy/dotnet-core